packages/happo-core/src/getImageFromDataURI.js

```diff
@@ -0,0 +1,15 @@
+const parseDataUri = require('parse-data-uri');
+const { PNG } = require('pngjs');
+
+function getImageFromDataURI(uri) {
+  return new Promise((resolve, reject) => {
+    const img = parseDataUri(uri);
+    const png = new PNG();
+    png.parse(img.data, (err) => {
+      if (err) reject(err);
```

Comment:
The Airbnb styleguide [only requires braces with multi-line blocks](https://github.com/airbnb/javascript#blocks)